cmd/fleet/handleCheckin.go

```diff
@@ -79,6 +81,7 @@ type CheckinT struct {
 }
 
 func NewCheckinT(
+	verCon version.Constraints,
```

Comment:
are we always going to have the version constraint? 
if not not then make it option function?
nit: the number of params in the constructor "is too darn high" :-)

Reply:
yes we should always have it